Makefile

```diff
@@ -97,7 +97,7 @@ $(OPENSHIFT_GOIMPORTS):
 imports: $(OPENSHIFT_GOIMPORTS)
 	$(OPENSHIFT_GOIMPORTS) -m github.com/kcp-dev/kcp
 
-COUNT ?= 5
+COUNT ?= 1
```

Comment:
This disables support for cached test results, which some folks might want. Should I file a separate issue to discuss if we want to support that or not?

Reply:
Let's see if anyone else has opinions - @kcp-dev/kcp-prototypers?

Reply:
Updated to set continue always setting `-count` for e2e, since the dependency on external binaries doesn't really allow for caching. For unit tests, only set -count if an explicit value for COUNT is provided.

Reply:
It turns out that enabling `-coverprofile` (which we do) negates caching when `-count` is unset, e.g.:

```
go test -race ./pkg/crdpuller
ok      github.com/kcp-dev/kcp/pkg/crdpuller    (cached)

go test -race -coverprofile=coverage.txt ./pkg/crdpuller
ok      github.com/kcp-dev/kcp/pkg/crdpuller    0.653s  coverage: 24.7% of statements
```

So we either need to abandon this (for now?), or make coverage off by default.

Reply:
Arg. Open to either, which would you prefer?

Reply:
Let's keep coverage on by default for the time being, so I guess let's undo this bit if that's ok?

Reply:
Done